### PR TITLE
Move definition of PIDdT back to temperature.h

### PR DESCRIPTION
**Pull request by AnHardt**
make it unconditional
remove all definitions in the 'Configuration.h's
remove form 'Conditionals.h'

### CODE REVIEW

**Comment by AnHardt** [comment]
#1683

File: Marlin/temperature.cpp [hunk 1]
```diff
@@ -45,6 +45,10 @@
   #define K2 (1.0-K1)
 #endif
 
+#ifdef PIDTEMPBED
```
Comment (tonnico) [inline Marlin/temperature.cpp]: Not only for bedtemp?

**Comment by tonnico** [comment]
Maybe we don't need to test this. The #defines will only used when need?

But there should be an #undef befor to be compatible with some older configs.

**Comment by AnHardt** [comment]
This is true for a lot of #defines and macros in marlin.
I like the conditions because they give a good hint for what or when things are needed or used.

I removed the definition of PIDdT in all configurations we provide. An error or warning during compile may make the 'user' aware of a problem with his old Configuration.h. I think this is better than silently overwriting the defined value - macro and make the 'user' wonder why things do not work as expected.

**Comment by thinkyhead** [comment]
@AnHardt I agree, we should probably throw errors to encourage people to update their configs. Especially with recent changes cutting down those files so much.

**Pull request merged**